content/en/news/releases/1.14.x/announcing-1.14/change-notes/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Istio 1.14 Change Notes
+linktitle: 1.14.0
+subtitle: Minor Release
+description: Istio 1.14.0 change notes.
+publishdate: 2022-05-24
+release: 1.14.0
+weight: 10
+aliases:
+- /news/announcing-1.14.0
+---
+
+## Traffic Management
+
+- **Added** support for sending unready endpoints to Envoy. This will be useful when slow start mode in Envoy is enabled.
+  This can be disabled by setting `PILOT_SEND_UNHEALTHY_ENDPOINTS` to false.
+
+- **Added** new configuration options to `istio-iptables` and `istio-clean-iptables`
+  for including/excluding certain user groups from interception of the outgoing traffic
+  generated by them.
+
+  This feature is intended primarily for use on VMs, where system administrators need
+  to restrain interception of the outgoing traffic down to a few applications instead
+  of intercepting all outgoing traffic.
+
+  By default, as before, the Istio Sidecar will intercept outgoing traffic from all processes,
+  no matter what user groups they are running under.
+
+  To change this behavior, system administrators can now use 2 new environment variables
+  supported by `istio-iptables` and `istio-clean-iptables` : `ISTIO_OUTBOUND_OWNER_GROUPS`
+  and `ISTIO_OUTBOUND_OWNER_GROUPS_EXCLUDE`.
+
+  `ISTIO_OUTBOUND_OWNER_GROUPS` is a comma separated list of groups whose outgoing traffic
```

Comment:
wondering why auto-san-validation is not showing up in change notes. Is it not included in the build? https://github.com/istio/istio/pull/39004

Reply:
https://github.com/istio/istio/pull/39004 was merged 13 days ago (May 18). The 1.14.0 build was created on May 27.  https://github.com/istio/istio/pull/39004 is included in the build.